### PR TITLE
Fix /collection/:id integration test

### PR DESCRIPTION
**Pull request by anandaroop**
Follow-up to https://github.com/artsy/force/pull/4614

This re-enables an integration test which was broken by https://github.com/artsy/reaction/pull/2845

There's interest in moving smoke tests like this over to https://github.com/artsy/integrity, but in the meantime I didn't want to lose coverage about the collection app being wired up correctly.

See further discussion in https://artsy.slack.com/archives/C2TQ4PT8R/p1570211335035500

### CODE REVIEW

File: src/test/acceptance/README.md [hunk 1]
```diff
@@ -28,10 +28,12 @@ describe('', () => {
       $('body').html().should.containEql('These Terms of Use')
     })
   })
-````
+```
 
 ## Fixtures
 
+_**Note:** The description below appears to be obsolete. See [this thread](https://artsy.slack.com/archives/C2TQ4PT8R/p1570211335035500) in the meantime for a workaround for capturing fixtures._
+
```
Comment (damassi) [inline src/test/acceptance/README.md]: 👍

**Comment by damassi** [comment]
Looks good! 

merge on green

File: src/test/acceptance/README.md [hunk 2]
```diff
@@ -28,10 +28,12 @@ describe('', () => {
       $('body').html().should.containEql('These Terms of Use')
     })
   })
-````
+```
 
 ## Fixtures
 
+_**Note:** The description below appears to be obsolete. See [this thread](https://artsy.slack.com/archives/C2TQ4PT8R/p1570211335035500) in the meantime for a workaround for capturing fixtures._
```
Comment (anandaroop) [inline src/test/acceptance/README.md]: Could probably be more aggressive in cutting stale docs here — I just was not really familiar with the state of this `acceptance-record` code (other than that it didn't work for me 😛 )

File: src/desktop/components/frame_animator/test/index.jest.js [hunk 3]
```diff
@@ -1,45 +1,46 @@
-import sinon from "sinon"
+import FrameAnimator from "../index"
 
-const rewire = require("rewire")("../index")
-const FrameAnimator = rewire.default
```
Comment (anandaroop) [inline src/desktop/components/frame_animator/test/index.jest.js]: This (unrelated) spec started breaking on this test run as well, with a `rewire` related error

Converting it to Jest and getting rid of the rewire stuff was sufficient to get this green again.

**Pull request merged**